pom.xml

```diff
@@ -133,49 +133,49 @@
         <dependency>
             <groupId>${project.groupId}</groupId>
             <artifactId>blueocean-commons</artifactId>
-            <version>${project.version}</version>
+            <version>${project.parent.version}</version>
```

Comment:
I should probably revert this, but the issue was if you use the blueocean parent (which I was trying for another reason) you have to set your plugin to the same as the blueocean verasion. But one shouldn't need to use the blueocean parent, so...